src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -523,7 +523,9 @@ function showNewQuestionFrame(type) {
     $('#empty_message').hide();
     scrollToElement($('#questionTableNew')[0], { duration: 1000 });
     $('#questionTableNew').find('.visibilityOptions').hide();
-    getVisibilityMessage($('#questionTableNew').find('.visibilityMessageButton'));
+
+    $newQuestionForm = $('#questionTableNew').closest('form');
+    updatePreviewTab($newQuestionForm);
```

Comment:
May or may not be related, but isn't this procedure what `matchVisibilityOptionToFeedbackPath` does?